packages/manager/apps/pci-volume-snapshot/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
```

Comment:
I will suggest to take the opportunity to turn on the `strict` mode to fine tune ts engine :)